Application/EdFi.Ods.Api/Controllers/VersionController.cs

```diff
@@ -43,6 +43,15 @@ public class VersionController : ControllerBase
             _apiSettings = Preconditions.ThrowIfNull(apiSettings, nameof(apiSettings));
         }
 
+        [HttpGet]
+        [Route("GetYearsForYearSpecific")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public IActionResult GetYearsForYearSpecific()
+        {
+            var years = _apiSettings.YearSpecificYears;
+            return Ok(years);
+        }
+
```

Comment:
is adding a new endpoint something we discussed in refinement?

Reply:
@tbmeador-dlp - I don't we discussed an implementation for this one during refinement, just an acceptance criteria.  If you think it's best,  I can move the config setting to the swagger project and remove endpoint.